src/device/pf_cmina.c

```diff
@@ -21,24 +21,29 @@
  * Manages sending DCP hello requests.
  *
  * Resets the communication stack and user application.
+ *
+ * States are SETUP, SET_NAME, SET_IP and W_CONNECT.
+ *
  */
 
-
 #ifdef UNIT_TEST
 #define os_set_ip_suite mock_os_set_ip_suite
+#define os_load_blob mock_os_load_blob
+#define os_save_blob mock_os_save_blob
+#define os_clear_blob mock_os_clear_blob
 #endif
 
 #include <string.h>
 #include <ctype.h>
 
 #include "pf_includes.h"
 
-static const char             *hello_sync_name = "hello";
+static const char          *hello_sync_name = "hello";
 
 
 /**
  * @internal
- * Send a DCP HELLO message.
+ * Trigger sending a DCP HELLO message.
```

Comment:
This new text suggests (to me) that it simply sets a flag and then "somebody else" does the sending, but pf_dcp_hello_req actually sends the request.
I think we should keep the previous text.

Reply:
Yes, this text is for pf_cmina_send_hello().